### PR TITLE
add component and context for app

### PR DESCRIPTION
**Pull request by shivamSspirit**
add component and context for app library

### CODE REVIEW

**Review by sanjay-xdr — COMMENTED** [review]
Delete console log 
and remove comment
everything else looks good

File: src/components/cards/exploreVideoCard/cardIcon.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+                    <li onClick={()=>clicck('Backend Development')}  className={`cate-li ${globalStateproperties?.selectedsingleVideoCategory?.categoryName === 'Backend Development' && `active`}`}>Backend</li>
+                    <li onClick={()=>clicck('full stack Development')}  className={`cate-li ${globalStateproperties?.selectedsingleVideoCategory?.categoryName === 'full stack Development' && `active`}`}>fullstack</li>
+                    <li onClick={()=>clicck('web3')} className={`cate-li ${globalStateproperties?.selectedsingleVideoCategory?.categoryName === 'web3' && `active`}`}>Blockchanin</li>
+                </ul>
+            </div>
+            <div className='col-container'>
+                <OneCard openModal={openModal} setModal={setModal} isExploreVideoCard={true} exploreVideoData={(globalStateproperties?.selectedsingleVideoCategory ? filteredCategory : globalVideos)}/>
+                 {/* {(globalStateproperties?.selectedsingleVideoCategory ? filteredCategory : globalVideos)?.map((item, idx) => (
```
Comment (aurobindoGupta) [inline src/components/cards/exploreVideoCard/cardIcon.js:37]: remove

File: src/components/cards/exploreVideoCard/cardicon.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    gap: 1rem; 
+    flex-wrap: wrap;
+    background-color: rgb(72, 56, 59);
+    justify-items: center;
+    align-items: center;
+    justify-content: center;
+    overflow: scroll;
+}
+
+/* .card-container {
```
Comment (aurobindoGupta) [inline src/components/cards/exploreVideoCard/cardicon.css:54]: remove

File: src/components/cards/oneCard/oneCard.js [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+import { useNavigate } from 'react-router-dom'
+import { useWatchLater } from '../../../context/watchLaterContext'
+import { usePlayList } from '../../../context/playListContext'
+
+import * as CategoryPis from '../../../api/category'
+import * as WatchLaterApi from '../../../api/watchlater'
+import * as playListApis from '../../../api/playlist'
+import * as ActionTypes from '../../../constant/actions'
+
+// import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
```
Comment (aurobindoGupta) [inline src/components/cards/oneCard/oneCard.js:13]: remove

File: src/components/playlistmanagment/playlistform/playList.js [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+
+    // setting cuurent video to respected playlist
+
+    const setvideoToplaylist = async (playListID) => {
+        setChecks(!checks)
+        if (playList?.currentselectedVideo) {
+            const response = await playListAPis?.postSingleplayList(playListID, playList?.currentselectedVideo)
+            console.log('aftersend video to respected playlist', response)
+        }
+        // await dispatchplayList({
```
Comment (aurobindoGupta) [inline src/components/playlistmanagment/playlistform/playList.js:128]: remove

**Review by aurobindoGupta — APPROVED** [review]
nice work
don't push commented code in pr

**Pull request merged**